### PR TITLE
Fix for #264, using the tinymce init promise if available

### PR DESCRIPTION
**Pull request by moritzpflaum**
I made a plnkr showing the issue:
http://plnkr.co/edit/R7oHmGcRkKz5ZZXSFo9h

And one showing it fixed:
http://plnkr.co/edit/F1QJlRPN8M04n3yNCgm5

The tests fail because of the line 
beforeEach(module('ui.tinymce'));

which resets the variable generatedIds to zero before every test, resulting in duplicate ids in the test DOM. If it is replaced with something global, the tests pass. This should not happen in non-test use, I think. I just don't know how to tell jasmine to load the module only once.

### CODE REVIEW

**Comment by timelf123** [comment]
LGTM, can we merge this?

**Comment by Nickproger** [comment]
Really need it... Still waiting...

**Comment by hajderr** [comment]
This one is good. Waiting for it too.

**Comment by ivorobkalo** [comment]
also waiting

**Comment by mcblum** [comment]
Need this PR merged if possible!

**Comment by Edvardoh** [comment]
Waiting for this as well =)

**Comment by deeg** [comment]
Can you please squash this PR into one commit so I can get it merged in?

File: src/tinymce.js [hunk 1]
```diff
@@ -138,8 +138,12 @@ angular.module('ui.tinymce', [])
           if (options.baseURL){
             tinymce.baseURL = options.baseURL;
           }
-          tinymce.init(options);
-          toggleDisable(scope.$eval(attrs.ngDisabled));
+		  var maybeInitPromise = tinymce.init(options); // newer versions of tinymce return a promise
```
Comment (ohjames) [inline src/tinymce.js]: tabs at the beginning here, messes up the indentation.

**Comment by asaarnak** [comment]
@deeg i have squashed it in new pull request https://github.com/angular-ui/ui-tinymce/pull/287

**Pull request closed without merging**